pymtl3/passes/test/DynamicSchedulePass_test.py

```diff
@@ -11,7 +11,6 @@
 from pymtl3.dsl.errors import UpblkCyclicError
 from pymtl3.passes.DynamicSchedulePass import DynamicSchedulePass
 from pymtl3.passes.GenDAGPass import GenDAGPass
-from pymtl3.passes.rtlir.behavioral.test.BehavioralRTLIRL1Pass_test import XFAIL_ON_PY3
```

Comment:
Oh so these failed tests are fixed? Good job!

Reply:
It was about the `exec` which no longer writes to `locals()` in Python 3. My import tests need dynamic scheduling so I fixed that.